### ISSUE TITLE
filter should have a --oneof flag

### ISSUE THREAD

**Issue by catdad** [catdad/json-cli-toolkit#25]
```bash
json filter -rim --attr name --oneof 1,2,3,4
json filter -rim --attr name --not --oneof 1,2,3,4
```

- [x] implement in filter tool
- [x] add filter unit tests
- [x] add to cli flag list
- [x] add cli/toolkit tests
- [x] add docs

**Comment by catdad** [comment]
I'm thinking about naming this `--in` instead. Less typing and more English.

**Comment by catdad** [comment]
Implemented: `json filter --attr propname --in one two three`

**Comment by catdad** [comment]
Published in 0.10.0.
